src/main/java/com/opentok/util/HttpClient.java

```diff
@@ -591,13 +612,21 @@ public String setBroadcastLayout(String broadcastId, BroadcastProperties propert
         }
         String type = properties.layout().getType().toString();
         String stylesheet = properties.layout().getStylesheet();
+        String screenshareLayout = null;
         if(StringUtils.isEmpty(type)) {
-            throw new RequestException("Could not set the layout. Either an invalid JSON or an invalid layout options.");
+            throw new RequestException("Could not set the layout. Either an invalid JSON or an invalid layout " +
+                    "99options.");
```

Comment:
Typo, though really I would just revert this change, it doesn't really do much
```suggestion
                    "options.");
```

Reply:
Looks like I accidentally added it.